Assets/HoloToolkit/Input/Tests/Scripts/DisplayKeywords.cs

```diff
@@ -0,0 +1,31 @@
+﻿using HoloToolkit.Unity;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DisplayKeywords : MonoBehaviour
+{
+    public KeywordManager keywordManager;
+    public Text textPanel;
+
+    void Start()
+    {
+        if (keywordManager == null)
+        {
+            Debug.Log("Please specify a KeywordManager in the Inspector on DisplayKeywords.cs on" + name + ".");
```

Comment:
I wouldn't want to return from the first one because it would never print the second message if both objects are null. Is there a better way to do it?